source/libraries/ContractDeployer.ts

```diff
@@ -22,21 +22,24 @@ import {
     DSRoles,
     DSValue,
     MatchingMarket,
-    GemPit
+    GemPit,
+    SaiTub,
+    SaiMom,
 } from "./ContractInterfaces";
 import BN = require("bn.js");
 
 type ContractAddressMapping = { [name: string]: string };
 
 const ETHER = new BN(10).pow(new BN(18));
+const MAX_APPROVAL = new BN("2").pow(new BN(256)).sub(new BN(1));
 
 export class ContractDeployer {
     private readonly accountManager: AccountManager;
     private readonly configuration: DeployerConfiguration;
     private readonly connector: Connector;
     private readonly contracts: Contracts;
 
-    private readonly defaultEthPriceInUsd   = "0x0000000000000000000000000000000000000000000000200000000000000000";
+    private readonly defaultEthPriceInUsd = "0x0000000000000000000000000000000000000000000000200000000000000000";
```

Comment:
A comment indicating what these two values are in base10 would be nice (certainly not required though).